cli/cmd/generate.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
```

Comment:
case-insensitive import collision

```suggestion
	"github.com/Sirupsen/logrus"
```

Reply:
Done